src/views/virtualmachines/details/tabs/snapshots/components/modal/SnapshotFormFields/SnapshotNameFormField.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { RedExclamationCircleIcon } from '@openshift-console/dynamic-plugin-sdk';
+import { FormGroup, TextInput } from '@patternfly/react-core';
+
+import { validateSnapshotName } from '../../../utils/helpers';
+
+type SnapshotNameFormFieldProps = {
+  snapshotName: string;
+  setSnapshotName: React.Dispatch<React.SetStateAction<string>>;
+  usedNames: string[];
+  setIsError: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const SnapshotNameFormField: React.FC<SnapshotNameFormFieldProps> = ({
+  snapshotName,
+  setSnapshotName,
+  usedNames,
+  setIsError,
+}) => {
+  const { t } = useKubevirtTranslation();
+
+  const [snapshotNameError, setSnapshotNameError] = React.useState(undefined);
+
+  const handleNameChange = (value: string, event: React.FormEvent<HTMLInputElement>) => {
+    event.preventDefault();
+    const error = validateSnapshotName(value, usedNames, t);
```

Comment:
we don't need this, the request should fail from the backend

Reply:
so no need for any validation? how about some trivial checks like that the input is not emtpy?

Reply:
no need, we don't want client side validation. only if its really required